README.md

```diff
@@ -184,6 +184,7 @@ _this is the recommended way to run remark42_
 | restricted-words        | RESTRICTED_WORDS        |                          | words banned in comments (can use `*`), _multi_ |
 | edit-time               | EDIT_TIME               | `5m`                     | edit window                                     |
 | read-age                | READONLY_AGE            |                          | read-only age of comments, days                 |
+| img-proxy               | IMG_PROXY               | `false`                  | enable http->https proxy for images (*deprecated*, please use image-proxy.http2https |
```

Comment:
I don't know if deprecated options should be exposed in documentation (they probably should with notification in which version they will be removed?), but I think that if you truly want to deprecate some function, show a warning in the log so the user would know them need to change something in their configuration.

Reply:
> I don't know if deprecated options should be exposed in documentation

I'd suggest adding another MD table "deprecated parameters" collapsed by default

> show a warning in the log so the user would know they need to change the configuration.

agree

Reply:
I'd suggest showing warnings in one version after deprecation (1.5.0 for now), don't use variable and still show information on how to fix it on the second version after deprecation (1.6.0 for us) and truly remove the option in next version (1.7.0 if we are starting with 1.5.0).

This will require some "check for deprecated options" code on our side and some effort on renewing all this stuff before new releases but would be very graceful for the users. That is the behavior I would expect from the mature project, and that I believe similar to how kubernetes handles deprecations.

Reply:
@smaant my changes are merged, you can move deprecated option to [appropriate section](https://github.com/umputun/remark#deprecated) in the Readme and write deprecated option handling in [HandleDeprecatedFlags](https://github.com/umputun/remark/blob/d0ddd8aa80615c20dad5ef3ad30589ac44f3d820/backend/app/cmd/server.go#L271) function.

Reply:
great, thank you, I've rebased and added `img-proxy` into this function